example-common/repository-jdbc/src/main/java/org/apache/shardingsphere/example/common/jdbc/service/CommonServiceImpl.java

```diff
@@ -17,60 +17,80 @@
 
 package org.apache.shardingsphere.example.common.jdbc.service;
 
+import org.apache.shardingsphere.example.common.entity.Country;
 import org.apache.shardingsphere.example.common.entity.Order;
 import org.apache.shardingsphere.example.common.entity.OrderItem;
+import org.apache.shardingsphere.example.common.repository.CountryRepository;
 import org.apache.shardingsphere.example.common.repository.OrderItemRepository;
 import org.apache.shardingsphere.example.common.repository.OrderRepository;
 import org.apache.shardingsphere.example.common.service.CommonService;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Locale;
 
 public final class CommonServiceImpl implements CommonService {
-    
+
     private OrderRepository orderRepository;
-    
+
     private OrderItemRepository orderItemRepository;
-    
+
+    private CountryRepository countryRepository;
```

Comment:
you can add a CountryServiceImpl  instead of modifying the commonServiceImpl like we have done in UserServiceImpl. other scenario as Sharding, transaction, orchestration don't need a broadcast-table.

Reply:
ok